pkg/api/types/v1/component.go

```diff
@@ -153,3 +160,66 @@ var MappingTableSchema = bigquery.Schema{
 		Type: bigquery.DateTimeFieldType,
 	},
 }
+
+type VariantMapping struct {
+	// APIVersion specifies the schema version, in case we ever need to make
+	// changes to the bigquery table that are not simple column additions.
+	APIVersion string `bigquery:"apiVersion"`
+
+	// Kind is a string value representing the resource this object represents.
+	Kind string `bigquery:"kind"`
+
+	// Product is the layer product name, to support the possibility of multiple
+	// component readiness dashboards. Generally leave this blank.
+	Product string `bigquery:"product"`
+
+	// JiraProject specifies the JIRA project that this variant belongs to.
+	JiraProject string `bigquery:"jira_project"`
+
+	// JiraComponent specifies the JIRA component that this variant belongs to.
+	JiraComponent string `bigquery:"jira_component"`
+
+	// VariantCategory defines the name of the variant
+	VariantCategory string `bigquery:"variant_category"`
+
+	// VariantValue defines the value of the variant
+	VariantValue string `bigquery:"variant_value"`
+
```

Comment:
I wasn't sure if Category is what we call this. I think Devan just calls it Variant and Value in the registry, but whatever it is we should just be consistent with sippy